README.md

```diff
@@ -25,7 +25,7 @@ Ensure the following tools are installed and present in your `$PATH`:
 
 - [`pulumictl`](https://github.com/pulumi/pulumictl#installation)
 - [Go 1.17](https://golang.org/dl/) or 1.latest
-- [NodeJS](https://nodejs.org/en/) 14.x.  We recommend using [nvm](https://github.com/nvm-sh/nvm) to manage NodeJS installations.
+- [NodeJS](https://nodejs.org/en/) 14.x or greater.  We recommend using [nvm](https://github.com/nvm-sh/nvm) to manage NodeJS installations.
```

Comment:
We actually aim to match the nodejs on their supported versions so should probably just say that. 

```suggestion
- [NodeJS](https://nodejs.org/en/) Active or maintenance version ([Node.js Releases](https://nodejs.org/en/about/previous-releases)).  We recommend using [nvm](https://github.com/nvm-sh/nvm) to manage NodeJS installations.
```